lib/engine/game/g_1829/game.rb

```diff
@@ -182,20 +137,27 @@ def all_potential_upgrades(tile, tile_manifest: false, selected_company: nil)
           upgrades
         end
 
+        def setup
+          corporations.each do |i|
```

Comment:
PAR_PRICES.each do |corporation, price|
  corporation = corporation_by_id(corporation)
  @stock_market.sett_par(corporation ...)

Reply:
can you just use this code?